deploy-help/configure.sh

```diff
@@ -0,0 +1,48 @@
+#! /bin/bash -e
+
+deployment_dir=~/deployment
+declare -A repositories
+
+repositories=(
+    ["concent-deployment"]="https://github.com/golemfactory/concent-deployment.git"
+    ["concent-deployment-values"]="https://github.com/golemfactory/concent-deployment-values.git"
+)
+
+if [ ! -d $deployment_dir ]; then
+    mkdir $deployment_dir/
+fi
+for repository in "${!repositories[@]}"; do
+    repository_path="$deployment_dir/$repository/"
+    if [ ! -d $repository_path ]; then
+        git clone "${repositories[$repository]}" "$repository_path"
+    else
+        cd "$repository_path"
+        git fetch origin --prune
+    fi
+done
+
+declare -A concent_dev_secrets_directories
+concent_dev_secrets_directories=(
+    ["concent-dev"]="$deployment_dir/concent-secrets/concent-dev/"
+    ["cloud"]="$deployment_dir/concent-secrets/cloud/"
+)
+
+if [ ! -d $deployment_dir/concent-secrets/ ]; then
+    mkdir $deployment_dir/concent-secrets/
+fi
+for secrets_directory in "${!concent_dev_secrets_directories[@]}"; do
+    if [ ! -d ${concent_dev_secrets_directories[$secrets_directory]} ]; then
+        cp                                                           \
+            --recursive                                              \
+            "/var/concent-data/concent-secrets/$secrets_directory/"  \
+            "${concent_dev_secrets_directories[$secrets_directory]}"
+    fi
+done
+
+echo "concent-builder ansible_connection=local" > $deployment_dir/inventory
```

Comment:
I'd just use `ansible_inventory` from concent-deployment-values and add `--connection=local` as a command-line parameter when running `ansible-playbook`. The less stateful configuration the better.